device/transport/http/src/http.ts

```diff
@@ -319,9 +403,8 @@ export class Http extends EventEmitter implements Client.Transport, BatchingTran
    */
   /*Codes_SRS_NODE_DEVICE_HTTP_16_001: [The ‘abandon’ method shall call the ‘abandon’ method of the receiver object and pass it the message and the callback given as parameters.] */
```

Comment:
>The ‘abandon’ method shall call the ‘abandon’ method of the receiver object [](start = 40, length = 75)

WTF?  The receiver object also did feedback?  Am I confused?

Reply:
yeah that was weird. not relevant anymore. i forgot to remove these requirements. fixing this.

---
In reply to: [143038637](https://github.com/Azure/azure-iot-sdk-node/pull/102#discussion_r143038637) [](ancestors = 143038637)